keyboards/matchstickworks/southpad/rev1/info.json

```
@@ -0,0 +1,55 @@
+{
+    "keyboard_name": "SouthPad",
+    "processor": "atmega32u2",
+    "bootloader": "atmel-dfu",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x6060",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["B0", "B1", "B2", "B4"],
+        "rows": ["B3", "B5", "B6", "B7", "C7", "C6"]
+    },
+    "layouts": {
+        "LAYOUT_numpad_6x4_southpaw": {
```

```suggestion
        "LAYOUT": {
```
Would you be able to also apply this change to `rev1/keymaps/default/keymap.c`